### ISSUE TITLE
Guest may Schedule a Meeting with a Team Member using a Public Schedule Link

### ISSUE THREAD

**Issue by zspencer** [zinc-collective/support#35]
```
Scenario: Guest may Schedule a Meeting with a Team Member using a Public Schedule Link
  Given a Team Member with a Public Schedule Link
  And a Guest is on the Team Member's Public Schedule Link
  When the Guest selects an Available Time Slot
  Then a Calendar Invite is sent to the Guest
  And a Calendar Invite is sent to the Team Member
```

**Comment by user512** [comment]
Catching up on this project, and I have feeling this ticket should be completed prior to https://github.com/zinc-collective/support/issues/13 because upon the completion of this ticket, public schedule link will be available.

Here is what I have in mind for this task:
1. Create a calendar-like UI for Guest to schedule meeting with team member
1. Upon scheduling, sent emails to both guest and team member with a `.ics` file.
1. We can create another ticket for real calendar integration, either thru web APIs or look into how [CalDAV](https://tools.ietf.org/html/rfc4791) works.

EDIT: After chatting with @maximegalon5, turns out it's more like an inverse scheduling. #13 should go first.

**Comment by zspencer** [comment]
Yes, our goal here is to make it so the public schedule link shows the spaces the team has made available for scheduling on a standard calendar view (similar to whentomeet, for example).

Once they've picked the time, we will want it to send an ics calendar invite via email to the team member participating and the person requesting the time. Ideally we'll be able to listen for cancellations/RSVP's of "No" on either end and keep everyone in the loop; but that's far far further in the future.

For now, we want to look for the _smallest_ steps we can take that are still _reasonable_ and potentially viable. 

The reason I sliced this out from #13 is because even if #13 isn't built, having a public schedule link is still useful if we create an `AvailableTimeSlot` using a a rake task or something.
